cmd/start.go

```diff
@@ -288,7 +289,10 @@ func runContainer(cmd *cobra.Command, args []string) {
 
 	log.Printf("Running cluster %s | image %s | flavor %s {%s Memory, %d CPU} ...", containerNameToShow, getImageName(), flavor, getMemorySize(flavor), ressources.NanoCPUs)
 
-	resp, err := getDocker().ContainerCreate(ctx, config, hostConfig, nil, containerName)
+	resp, err := getDocker().ContainerCreate(ctx, config, hostConfig, nil, &v1.Platform{
+		Architecture: "amd64",
```

Comment:
Do we want to pick arch/OS via env variables here? I've hardcoded them for now..